.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
- [ ] We should not commit unrelated files to our repository. I think it should be very nice if you removed this `.vscode` folder from your repository